ffmpeg_progress_yield/ffmpeg_progress_yield.py

```diff
@@ -60,6 +60,8 @@ def run_command_with_progress(self):
             stdout=subprocess.PIPE,
             stderr=subprocess.STDOUT,
             universal_newlines=False,
+            shell=False,
+            creationflags=CREATE_NO_WINDOW
```

Comment:
`CREATE_NO_WINDOW` is only available [from Python 3.7 upwards](https://docs.python.org/3/library/subprocess.html#subprocess.CREATE_NO_WINDOW) while this project targets Python 3.5 and higher.

Can you please explain what the behavior without vs. with this flag is? I am not a Windows user.

Reply:
In development mode this problem does not apply. 
After compiling the exe with pyinstaller, this window cmd is show while ffmpeg works.
I didn't know but breaking backwards compatibility is perhaps not a good idea.
The solution I got was this, maybe there is another way that is compatible with older versions.
Can consider this pull requests closed. You have a good job here, bye

Reply:
I think this is a valid use case. What I could do is that I could allow the user to pass extra options to the shell command, so if you are using this in your own tool, you can add that flag yourself. Does that make sense?

Reply:
Yes it makes sense. Parameterizing the inputs will be a good solution. Speaking of extra entries I remembered. I had some difficulty implementing it, looking your code then I found that there were overlapping commands. In run_command_with_progress function you have this:
```
cmd_with_progress = (
            [self.cmd[0]] + ["-progress", "-", "-nostats"] + self.cmd[1:]
        )
```
I modified it and I'm using it like this:

`cmd_with_progress = (
            [self.cmd[0]] + ["-progress", "-", "-nostats", "-hide_banner", '-stats_period', '0.2',] + self.cmd[1:]
        )`

using in the call this:

```
args = ['ffmpeg', '-c:a', 'aac', '-i', self.mp4cv, '-vn', '-ac', '2', '-ar', '48000', '-ab', '192', '-f', 'mp3', self.mp3cv]
 ff = FfmpegProgress(args)
```

Emphasizes here the idea that in your function there should be no parameter for ffmpeg, this will be all up to the user.
This is just my suggestion. Good job